src/js/view/controlbar.js

```diff
@@ -101,8 +107,8 @@ define([
             this.elements = {
                 alt: text('jw-text-alt'),
                 play: button('jw-icon-playback', this._api.play),
-                prev: button('jw-icon-prev', this._api.playlistPrev),
-                next: button('jw-icon-next', this._api.playlistNext),
+                prev: button('jw-icon-prev', this._api.playlistPrev, {reason: 'interaction'}),
```

Comment:
you can bind the arg, instead of adding a new param to "button"

```js
    prev: button('jw-icon-prev', this._api.playlistPrev.bind(this, {reason: 'interaction'}))
```